OpenRA.Game/Traits/Player/Shroud.cs

```diff
@@ -205,6 +209,17 @@ void ITick.Tick(Actor self)
 					var puv = touched.PPosFromIndex(index);
 					if (map.Contains(puv))
 						OnShroudChanged(puv);
+
+					if (fogEnabled || !ExploreMapEnabled)
```

Comment:
```suggestion
					if (!disabledChanged && (fogEnabled || !ExploreMapEnabled))
```

This will fix the debug bug. When we use the cheat we refresh all cells and the current code picks it up as new cells being added

Reply:
Fix confirmed and it doesn't break the rest. Thanks!